### PR TITLE
KS: Removal of the least used macros - l10n-ko

### PR DESCRIPTION
**Pull request by tristantheb**
### 🎫 Related issues
See main PR : #611
mdn/yari#3063

### 👀 Observed issues
KumaScript macros are being phased out, to help the Yari dev team, here is a list of macros that will disappear from your files.

I invite you to check your files in your languages to make sure it doesn't break any translations. You can edit them directly on my branch. If you want to take care of it after merging this request, feel free :)

### 🔁 Changes in commits
*This list is taken from the issue quoted above, the numbers are not current.*
```
kumascript/macros/原語併記.ejs                         88
kumascript/macros/HTML5ArticleTOC.ejs              77
kumascript/macros/EditorGuideQuicklinks.ejs        42
kumascript/macros/gecko_callout_heading.ejs        38
kumascript/macros/IRCLink.ejs                      26
kumascript/macros/ref.ejs                          25
kumascript/macros/gecko_minversion_note.ejs        22
kumascript/macros/h1_gecko_minversion.ejs          20
kumascript/macros/FontAwesomeIcon.ejs              11
kumascript/macros/endnote.ejs                      11
kumascript/macros/h2_gecko_minversion.ejs          10
kumascript/macros/UserLink.ejs                     9
kumascript/macros/訳語.ejs                           6
kumascript/macros/LXRSearch.ejs                    2
kumascript/macros/tree.ejs                         2
kumascript/macros/B2GOnlyHeader2.ejs               1
```

```diff
! Only macros used less than 100 times for the moment.
! The pull request would be too big to process everything at once.
```

The compatibility tables are not done in this pull request (except when I stumble upon them). They require a query of their own, as there are many of them.

### CODE REVIEW

**Comment by tristantheb** [comment]
## Special note
You have time to do this, it is not an emergency at the moment. You can also do the other macros, this branch is open for modification.
The branch is also open to the entire community of the relevant language, so feel free to add as a reviewer.

File: files/ko/web/html/element/textarea/index.html [hunk 1]
```diff
@@ -97,8 +97,6 @@ <h2 id="DOM_인터페이스">DOM 인터페이스</h2>
 
 <p>이 요소는 <code><a href="/en/DOM/HTMLTextAreaElement" title="en/DOM/textarea">HTMLTextAreaElement</a></code> 인터페이스를 구현합니다.</p>
 
-<p>{{ h1_gecko_minversion("Resizable textareas", "2.0") }}</p>
-
 <p>{{ gecko("2.0") }} introduced support for resizable textareas. This is controlled by the {{ cssxref("resize") }} CSS property. Resizing of textareas is enabled by default, but you can explicitly disable it by including the following CSS:</p>
```
Comment (hochan222) [inline files/ko/web/html/element/textarea/index.html:100]:
```suggestion
<p>Gecko 2.0 introduced support for resizable textareas. This is controlled by the <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/resize">resize (en-US)</a> CSS property. Resizing of textareas is enabled by default, but you can explicitly disable it by including the following CSS:</p>
```
> KumaScript macros are being phased out, to help the Yari dev team, here is a list of macros that will disappear from your files.

Could this be the case with macros like this?

File: files/ko/web/api/window/open/index.html [hunk 2]
```diff
@@ -76,8 +76,6 @@ <h2 id="Window_features">Window features</h2>
 
 <h3 id="Position_and_size_features">Position and size features</h3>
 
-<div>{{gecko_minversion_note("1.9.2", "Starting in Gecko 1.9.2 (Firefox 3.6), overriding the position of a window using window features will not change the persisted values saved by the session store feature. That means the next time the window is opened, it will still open in the saved location.")}}</div>
-
 <p><a href="#Note_on_position_and_dimension_error_correction">Note on position and dimension error correction</a></p>
 
 <div class="bug">{{bug(176320)}}</div>
```
Comment (hochan222) [inline files/ko/web/api/window/open/index.html:81]:
```suggestion
<div class="bug"><a href="https://bugzilla.mozilla.org/show_bug.cgi?id=176320">bug(176320)</a></div>
```

**Review by hochan222 — APPROVED** [review]
LGTM :) 
@tristantheb  I understood well. Thank you, I will make a plan in Korean locale and proceed with it.

**Comment by hochan222** [comment]
In order to get out of the risk of collision, to receive PR each time we create an issue separately, we plan to receive and process individual PRs, not only the above item macro but also other macro processing.

**Pull request merged**